posthog/admin.py

```diff
@@ -77,6 +80,38 @@ def organization_link(self, team: Team):
         )
 
 
+class PluginAttachmentInline(admin.StackedInline):
+    extra = 0
+    model = PluginAttachment
+    fields = ("key", "content_type", "file_size", "raw_contents", "json_contents")
+    readonly_fields = fields
+
+    def raw_contents(self, attachment: PluginAttachment):
+        try:
+            if attachment.file_size > 1024 * 1024:
+                raise ValueError("too big")
```

Comment:
nit: I'd prefer a more detailed error message, e.g. `ValueError("attachment file size {size} is larger than maximum preview limit {limit}")`

Reply:
<img width="760" alt="image" src="https://user-images.githubusercontent.com/6241083/231188558-36868007-f411-44a7-a47f-b155ba200260.png">

Reply:
Thank you!